pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -183,6 +183,33 @@ func StartLibvirt(stopChan chan struct{}) {
 	// container.
 
 	go func() {
+		// libvirtd needs access to /dev/null and /dev/urandom
+		// Ensure both are there and ready to be used, panic if they are still not after 1 second
+		devNodes := []string{"/dev/null", "/dev/urandom"}
+		for i := 1; i <= 10; i++ {
+			var err error
+			var fd *os.File
+			for _, node := range devNodes {
+				fd, err = os.Open(node)
```

Comment:
Do we really need to open the devices? Wouldn't os.Stat be enough to prove the endpoint exists? (Is there really a case where a null/urandom device node can exist and not be ready?)

Reply:
Yes actually that seems the be the case (node present but won't open). The error is `operation not permitted`.